build/android/bytecode/java/org/brave/bytecode/BraveBookmarkUtilsClassAdapter.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.brave.bytecode;
+
+import org.objectweb.asm.ClassVisitor;
+
+public class BraveBookmarkUtilsClassAdapter extends BraveClassVisitor {
+    static String sBookmarkUtilsClassName = "org/chromium/chrome/browser/bookmarks/BookmarkUtils";
+    static String sBraveBookmarkUtilsClassName =
+            "org/chromium/chrome/browser/bookmarks/BraveBookmarkUtils";
+
+    public BraveBookmarkUtilsClassAdapter(ClassVisitor visitor) {
+        super(visitor);
+        changeMethodOwner(
+                sBookmarkUtilsClassName, "addOrEditBookmark", sBraveBookmarkUtilsClassName);
```

Comment:
Please add checks in `testClassesExist` and `testMethodsExist` in `brave/android/javatests/org/chromium/chrome/browser/BytecodeTest.java`

Reply:
And whitelist it for tests here `brave/android/java/apk_for_test.flags`